src/outages.jl

```diff
@@ -0,0 +1,63 @@
+abstract type Outage <: SupplementalAttribute end
```

Comment:
@scdhulipala please confirm with @GordStephen that this structure is adequate for the purposes of integrating PSY into PRAS.

Reply:
As far as information needed, l think this looks good. Since there's multiple ways to store the same data, will there be a convention for which takes priority, or is that decision left to the consumer? E.g. FOR and MTTR won't necessarily be internally consistent with outage and recovery transition probabilities, or may be provided as time series vs static values. Also, seems like TimeSeriesContainer is pretty flexible as far as contents, will there be guidance around canonical names for the different possible time series stored within (presumably they'll be expected to match the struct property names)?

Reply:
As usual in PSY the data is left to the user. We can decide to limit the fields to one or the other if the additional flexibility doesn't help. I'll proceed to merge this branch into the PSY4 release branch to make further changes in a forthcoming PR.